### PR TITLE
[MVP-1694] remove SMS destination section

### PR DESCRIPTION
**Pull request by marukohao**
Test:
Screenshot 2022-12-01 at 11 31 39 AM
Screenshot 2022-12-01 at 11 32 49 AM

Story:
https://notifi.atlassian.net/browse/MVP-1694

### CODE REVIEW

**Review by jennifer-notifi — COMMENTED** [review]
Great job on consolidating classNames. Can you please also create a ticket for removing SMS in the Intercom Card in the config tool as well?

File: packages/notifi-react-card/lib/components/intercom/NotifiIntercomFTUNotificationTargetSection.tsx [hunk 1]
```diff
@@ -66,41 +64,6 @@ export const NotifiIntercomFTUNotificationTargetSection: React.FC<
           </div>
         </div>
       ) : null}
-      {data.contactInfo.sms.active ? (
-        <NotifiSmsInput
-          disabled={false}
-          classNames={classNames?.NotifiSmsInput}
-          allowedCountryCodes={allowedCountryCodes}
-          copy={{ label: inputLabels?.sms }}
-          intercomSmsInputContainerStyle={'NotifiIntercomSmsInput__container'}
```
Comment (jennifer-notifi) [inline packages/notifi-react-card/lib/components/intercom/NotifiIntercomFTUNotificationTargetSection.tsx:75]: Can we please also remove related smsInput classnames in the `default.css`?

Comment (marukohao) [reply packages/notifi-react-card/lib/components/intercom/NotifiIntercomFTUNotificationTargetSection.tsx:75]: good catch and ticket to remove sms in config tool has been created

**Review by jennifer-notifi — CHANGES_REQUESTED** [review]
Thank you, I did another quick run through of the code base. Can you please check on this as well?

- Remove any SMS error messages in `packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx`

**Comment by marukohao** [comment]
> Thank you, I did another quick run through of the code base. Can you please check on this as well?
> 
> * Remove any SMS error messages in `packages/notifi-react-card/lib/components/intercom/IntercomCard.tsx`

As Juni's suggestion, instead of removing the sms section in intercom card, we could keep the code for flexibility and prevent tenant turning on the sms toggle in config tool. I’ll close this PR.

**Pull request closed without merging**